pandas/util/testing.py

```diff
@@ -868,9 +868,9 @@ def assertIsInstance(obj, cls, msg=''):
     """Test that obj is an instance of cls
     (which can be a class or a tuple of classes,
     as supported by isinstance())."""
-    assert isinstance(obj, cls), (
-        "%sExpected object to be of type %r, found %r instead" % (
-            msg, cls, type(obj)))
+    if not isinstance(obj, cls):
+        err_msg = "{0}Expected type {1}, found {2} instead"
+        raise AssertionError(err_msg.format(msg, cls, type(obj)))
```

Comment:
Yeah. 
How dependent is the testing suite on nosetests, vs just unittest? 

We were using nosetests, moved to pytest, and it worked _really_ seamlessly. But that's because we were really only using nosetests as the runner, rather than any nose-specific features